modules/ossm-installing-multi-primary-multi-network-topology.adoc

```diff
@@ -0,0 +1,294 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-installing-multi-primary-multi-network-topology_{context}"]
+= Installing a multi-primary multi-network topology 
+:context: ossm-installing-multi-primary-multi-network-topology
+
+Install {istio} in a multi-primary multi-network topology on two {ocp-product-title} clusters. 
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster. 
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator on all of the clusters that comprise the mesh.
+
+* You have completed _Creating certificates for a multi-cluster mesh_. 
+
+* You have completed _Applying certificates to a multi-cluster topology_.
+
+* You have `istioctl` installed on the laptop you will use to run these instructions.
+
+.Procedure
+
+. Create an `ISTIO_VERSION` environment variable that defines the {istio} version to install by running the following command:
++
+[source,terminal]
+----
+$ export ISTIO_VERSION=1.24.0 
+----
+
+. Install {istio} on the East cluster:
+
+.. Create a YAML file named `istio-east.yaml` that defines an `Istio` resource on the East cluster.
++
+.Example resource file
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v${ISTIO_VERSION}
+  namespace: istio-system
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: cluster1
+      network: network1
+----
+
+.. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc --context "${CTX_CLUSTER1}" apply -f istio-east.yaml
+----
+
+.. Wait for the control plane to return the "Ready" status condition by running the following command:
++
+[source,terminal]
+----
+$ oc wait --context "${CTX_CLUSTER1}" --for=condition=Ready istio/default --timeout=3m
+----
+
+.. Create an East-West gateway on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply --context "${CTX_CLUSTER1}" -f https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/docs/multicluster/east-west-gateway-net1.yaml
+----
+
+.. Expose the services through the gateway by running the following command:
++
+[source,terminal]
+----
+$ oc --context "${CTX_CLUSTER1}" apply -n istio-system -f https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/docs/multicluster/expose-services.yaml
+----
+
+. Install a remote secret on the East cluster that provides access to the API server on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ istioctl create-remote-secret \
+  --context="${CTX_CLUSTER2}" \
+  --name=cluster2 | \
+  oc apply -f - --context="${CTX_CLUSTER1}"  
+----
```

Comment:
I believe this command must be run after both control planes are created (i.e. step "4. Install Istio on the West cluster" must be completed before step "3. Install a remote secret on the East cluster").

Also, I'd compress this command into two lines only (this actually runs two commands, so having each on its own line makes this easier to comprehend).
```suggestion
. Install a remote secret on the East cluster that provides access to the API server on the West cluster by running the following command:
+
[source,terminal]
----
$ istioctl create-remote-secret --context="${CTX_CLUSTER2}" --name=cluster2 | \
     oc apply -f - --context="${CTX_CLUSTER1}"  
----
```

Reply:
@luksa  I believe I made both of these changes correctly, but I would appreciate if you could just double-check.

Now, the steps to install a remote secret on the East cluster that provides access to the API server on the West cluster and to install a remote secret on the West cluster that provides access to the API server on the East cluster are one after the other, and both are after the step to Install Istio on the West cluster.